.github/workflows/pull-request-workflow.yml

```diff
@@ -7,9 +7,11 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [16.x]
+        node-version: [16.x, 18.x]
```

Comment:
Okay, instead to say where exactly we need to use matrix nodejs, let me try to explain the reason to having one nodejs version for one job and several versions for another. Let's think about is `format` job depends on nodejs version? I guess not, cuz it fully depends on prettier version which is easily managing. But what about build and tests jobs? I think it fully depends on nodejs version, moreover it depends on which nodejs version our library supports, so it's very important to check it on that job. So, instead blinding fixing each place manually as soon as I leave a new comment, try to figure out what do you need in advance without waiting my explicit comment

Reply:
Thanks for this information, I didn't know about it, my bad.